### PR TITLE
Add bioinformatics kmers example

### PR DESCRIPTION
**Pull request by ehsantn**
## Changes included in this PR

Adds a bioinformatics example that reads "fasta" files and combines the data into a CSV file.

## Testing strategy

Tested it manually. Will be tested as part of nightly example tests.

## User facing changes

A new example.

## Checklist
- [N/A] Pipelines passed before requesting review. To run CI you must include `[run CI]` in your commit message.
- [x] I am familiar with the [Contributing Guide](https://github.com/bodo-ai/Bodo/blob/main/CONTRIBUTING.md) 
- [x] I have installed + ran pre-commit hooks.

### CODE REVIEW

File: examples/generate_kmers_bodo.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import glob
```
Comment (scott-routledge2) [inline examples/generate_kmers_bodo.py:1]: Do we need a comment here to explain this example?

**Review by scott-routledge2 — APPROVED** [review]
LGTM, thanks @ehsantn !

File: examples/generate_kmers_bodo.py (hunk 1 above)
Comment (ehsantn) [reply examples/generate_kmers_bodo.py:1]: I don't have enough context. @marquisdepolis could you suggest a short description for the example?

**Comment by marquisdepolis** [comment]
Ran great for me, and was faster even with the smaller fasta files. Approving PR, and will check on publicising.

**Review by marquisdepolis — APPROVED** [review]

File: examples/generate_kmers_bodo.py (hunk 1 above)
Comment (ehsantn) [reply examples/generate_kmers_bodo.py]: Thanks, added.

**Pull request merged**